src/styles/root.scss

```diff
@@ -98,6 +98,10 @@ input {
 		}
 	}
 
+	&__embedded {
+		background-color: #fff;
```

Comment:
create new color variable in `src/styles/common/_colors.scss` for embeeded layout and use it